examples/cc3220/README.md

```diff
@@ -0,0 +1,134 @@
+Installing Dependencies & Related Documentation
+===============================================
+
+Context:
+
+- Hardware: CC3220SF-LaunchXL
+- SDK Version: simplelink_cc32xx_sdk_1_30_01_03
+- Computer OS: Windows 10
+- CCS Version: 7.0.0.00042
+- Compiler: ti-cgt-arm_16.9.0.LTS
+
+Dependencies:
+
+1. Install Code Composer Studio
+2. Install the [SimpleLink CC3220 SDK](http://www.ti.com/tool/simplelink-cc3220-sdk)
+3. Install [UniFlash v4](http://ti.com/tool/uniflash)
+
+Documentation for similar projects:
+
+- [Compiling CC3200 with Enhanced Security](https://developer.xively.com/v1.0/docs/ti-cc3200-advanced)
+
+Compiling the Firmware With Custom WolfSSL
+==========================================
+
+1. Clone xively-client-c into your Windows machine
+2. Export paths to gmake in Windows' `cmd`
+    ```
+    PATH=%PATH%;c:\ti\ccsv7\utils\bin
+    PATH=%PATH%;c:\ti\ccsv7\utils\cygwin
+    ```
+3. [Download WolfSSL source](https://developer.xively.com/v1.0/docs/ti-cc3200-advanced#section-download-wolfssl-library-source).
+Follow the steps in that tutorial to unzip and rename the WolfSSL folder
+4. Clean libxively from `cmd`
+    ```
+    $ cd $PATH_TO_XIVELY_CLIENT_C
+    $ gmake PRESET=CC3220 clean
+    ```
+5. Build libxively from `cmd`
+    ```
+    $ gmake PRESET=CC3220                                         \
+        XI_CC3220_PATH_SDK=C:\ti\simplelink_cc32xx_sdk_1_30_01_03 \
+        XI_CC3220_PATH_XDC_SDK=C:\ti\xdctools_3_50_01_12_core     \
+        XI_CC3220_PATH_CCS_TOOLS=C:\ti\ccsv7\tools                \
+        COMPILER=ti-cgt-arm_16.9.0.LTS                            \
+        BSP_DEBUG_LOG=1 XI_DEBUG_OUTPUT=1
+    ```
+6. Create a Code Composer Studio Workspace
+(e.g. `$PATH_TO_XIVELY_CLIENT_C/examples/cc3220/ccs_workspace`)
+7. `Project->Import CCS Project...` `$PATH_TO_XIVELY_CLIENT_C/examples/cc3220/xively_demo`
+8. `Project->Import CCS Project...` `$PATH_TO_XIVELY_CLIENT_C/examples/cc3220/wolfssl_ccs`
+9. `Project->Import CCS Project...` `C:\ti\simplelink_cc32xx_sdk_1_30_01_03\kernel\tirtos\builds\CC3220SF_LAUNCHXL\release\ccs`
+10. Update all necessary system paths in the `XivelyExample` and `wolfssl_ccs`
+project settings
+11. Build the `wolfssl_ccs` project
```

Comment:
Generally we're very specific with our tutorials, telling the user the exact build steps to do.  For instance, how does one Build the Wolfssl_ccs project?  

See the very end of https://developer.xively.com/docs/ti-cc3200-advanced for an example of how indepth we get with these instructions.